sdk/storage/storage-blob/test/blobserviceclient.spec.ts

```diff
@@ -5,6 +5,11 @@ import { BlobServiceClient } from "../src/BlobServiceClient";
 import { getAlternateBSU, getBSU, getUniqueName, wait } from "./utils";
 dotenv.config({ path: "../.env" });
 
+// async iterator polyfill for Node v8.
+if (Symbol["asyncIterator"] === undefined) {
+  (Symbol as any).asyncIterator = Symbol.asyncIterator || Symbol.for("Symbol.asyncIterator");
+}
```

Comment:
Just a thought - 

Can we avoid duplication by somehow adding this polyfill in the "core-paging" module itself ?